music21/stream/base.py

```diff
@@ -6682,6 +6682,39 @@ def makeBeams(self, *, inPlace=False, setStemDirections=True, failOnNoTimeSignat
             failOnNoTimeSignature=failOnNoTimeSignature,
         )
 
+    def makeOrnamentalAccidentals(
+        self,
+        noteOrChord: note.Note | chord.Chord,
+        *,
+        pitchPast: list[pitch.Pitch] | None = None,
+        pitchPastMeasure: list[pitch.Pitch] | None = None,
+        otherSimultaneousPitches: list[pitch.Pitch] | None = None,
+        alteredPitches: list[pitch.Pitch] | None = None,
+        cautionaryPitchClass: bool = True,
+        cautionaryAll: bool = False,
+        overrideStatus: bool = False,
+        cautionaryNotImmediateRepeat: bool = True,
+    ):
+        for orn in noteOrChord.expressions:
```

Comment:
-1:  undocumented, untested new addition to Stream.  Can this be documented and moved to stream.makeNotation....  -- I'm trying to trim down the complexity of the Stream class (and length of this file) and this seems highly unlikely for users to call on their own.  makeNotation.py is where the guts of `Stream.makeNotation()` needs to live.  I have as a todo to move makeBeams, makeAccidentals, etc. to makeNotation.py as well, but because they've appeared in demos, papers, etc. they're harder to move.

Reply:
Moved, and documented (but no doctests).  This routine is actually well covered by the test suite; do you also want doctests?

Reply:
Looks like both moved routines (`makeOrnamentalAccidentals` and `ornamentalPitches(s)` have 100% coverage.

Reply:
Great -- no doctests needed if it's well tested and well documented.  Thanks for moving it.